local_planner/src/nodes/local_planner_node.h

```diff
@@ -53,6 +54,38 @@ struct cameraData {
   bool received_;
 };
 
+
+
+struct ModelParameters {
+  float ekf2_rng_a_hmax = NAN;
+  float ekf2_rng_a_vmax = NAN;
+  float mc_pitchrate_max = NAN; // altitude control
+  float mc_rollrate_max = NAN;
+  float mc_yawrate_max = NAN;
+  float mc_yawauto_max = NAN;
+  float mpc_acc_down_max = NAN; // Maximum vertical acceleration in velocity controlled modes down
+  float mpc_acc_hor = NAN; // Maximum horizontal acceleration for auto mode and maximum deceleration for manual mode
+  float mpc_acc_up_max = NAN;  // Maximum vertical acceleration in velocity controlled modes upward
+  float mpc_col_prev_d = NAN; // Minimum distance the vehicle should keep to all obstacles
+  float mpc_hold_max_xy = NAN; // Maximum horizontal velocity for which position hold is enabled (use 0 to disable check)
+  float mpc_hold_max_z = NAN; // Maximum vertical velocity for which position hold is enabled
+  float mpc_jerk_max = NAN;
+  float mpc_jerk_min = NAN;
+  int mpc_pos_mode = NAN; //smooth position velocity  mpc_pos_mode = 3
+  float mpc_thr_max = NAN;
+  float mpc_thr_min = NAN;
+  float mpc_tiltmax_air = NAN;
+  float mpc_tko_speed = NAN; // Takeoff climb rate
+  float mpc_xy_cruise = NAN; // Maximum horizontal velocity in mission, capped by mpc_xy_vel_max
+  float mpx_xy_vel_max = NAN; // Maximum horizontal velocity in AUTO mode. 
+  float mpc_z_vel_max_dn = NAN;
+  float mpc_z_vel_max_up = NAN;
+
+
```

Comment:
Lots of white space issues in many places. Run `./tools/fix_style.sh local_planner` please. (This should also be caught by C.I.)